### PR TITLE
Room lobby modal CSS cleanup

### PR DESCRIPTION
**Pull request by gfodor**
It now looks like this:

image

### CODE REVIEW

File: src/react-components/ui-root.js [hunk 1]
```diff
@@ -1020,7 +1023,7 @@ class UIRoot extends Component {
                     e.target.blur();
                   }
                 }}
-                placeholder="Send to room..."
+                placeholder="Send message to others..."
```
Comment (johnshaughnessy) [inline src/react-components/ui-root.js]: Hmm... this copy feels awkward to me but am not sure if I can think of something better (that also fits on small screens). 
How about "Message the room..."?

File: src/react-components/ui-root.js [hunk 2]
```diff
@@ -960,6 +960,9 @@ class UIRoot extends Component {
 
     return (
       <div className={entryStyles.entryPanel}>
+        <div className={entryStyles.header}>
+          <FormattedMessage id="entry.room" />
```
Comment (johnshaughnessy) [inline src/react-components/ui-root.js]: I don't think this is a good addition

**Comment by gfodor** [comment]
PR is updated to improve responsiveness of the chatbox

**Comment by gfodor** [comment]
latest design:

image

**Pull request merged**